### PR TITLE
Simulation startup

### PR DESCRIPTION
**Pull request by ryankarpuszka**
Creates a `run_simulation.py` script to run all components of the simulation in one command. Also adds some command line arguments to `run_gui.py`.

### CODE REVIEW

**Comment by ryankarpuszka** [comment]
To get SITL working on a virtual environment I had to modify the setup script for ArduSub. To get these changes, cd into `ardupilot`. Run `git remote set-url origin git@github.com:CWRUbotixROV/ardupilot.git`
`git pull`
`workon rov`
`Tools/environment_install/install-prereqs-ubuntu.sh -y`

**Review by NoahMollerstuen — COMMENTED** [review]
Looks good, this will make managing different configurations a lot easier. I haven't actually tested it on my machine yet because of some file path stuff (see comments).

File: util.py [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
 parent_dir = path.split(path.dirname(__file__))[0]
 data_path = path.join(parent_dir, "data")
+ardusub_path = path.join(parent_dir, 'ardupilot', 'ArduSub')
+gazebo_path = path.join(parent_dir, 'gazebo_rov')
```
[the remark this answers is not in the document]
Comment (ryankarpuszka) [reply util.py:9]: That could be done. I just don't want to have too many config files that make it annoying to set up. Maybe we could make these the default path, but have some way to override it?

**Comment by ryankarpuszka** [comment]
@NoahMollerstuen I changed the config file in the repo to be a "default" config file, and then people who need to edit can copy it which will be ignored.

**Review by NoahMollerstuen — APPROVED** [review]
Someone else should probably review this now that I've contributed to it, but it looks good to me

**Comment by ryankarpuszka** [comment]
> Someone else should probably review this now that I've contributed to it, but it looks good to me

Whoops, I did not read this before merging.

**Comment by NoahMollerstuen** [comment]
> Whoops, I did not read this before merging.

All good lol. We reviewed each other's code so I think we're good.

**Pull request merged**